Neos.Neos/Classes/Command/WorkspaceCommandController.php

```diff
@@ -359,16 +367,18 @@ public function listCommand(string $contentRepository = 'default'): void
         }
 
         $tableRows = [];
-        $headerRow = ['Name', 'Base Workspace', 'Title', 'Owner', 'Description', 'Status', 'Content Stream'];
+        $headerRow = ['Name', 'Classification', 'Base Workspace', 'Title', 'Description', 'Status', 'Content Stream'];
```

Comment:
so we dont want to show the user id? Well i guess that makes sense as its too technical anyway and the human readable name with classification personal will already suffice?

Reply:
I'll certainly re-add it, but with the new Workspace <-> Role assignment, maybe rather in some new `showCommand()` that lists all contributors and their role